src/ctap/mod.rs

```diff
@@ -501,6 +522,7 @@ where
                     .user_display_name
                     .map(|s| truncate_to_char_boundary(&s, 64).to_string()),
                 cred_random,
+                cred_protect_policy: cred_protect_policy.clone(),
```

Comment:
I'm surprised we need this `clone`. Rust should deduce `Option<Policy>: Copy` from `Policy: Copy`.

Reply:
I think we needed before it implemented Copy, and I forgot to remove the clone. Fixed.